docs/concepts/index.md

```diff
@@ -28,7 +28,7 @@ tags:
 | Keyword                                                  | Description                                                                                        |
 | -------------------------------------------------------- | -------------------------------------------------------------------------------------------------- |
 | __[Deployments](/concepts/deployments)__                 | A server-side concept that encapsulates a flow, allowing it to be scheduled and triggered via API. |
-| __[Projects](/concepts/projects)__                       | A minimally opinionated set of files that describe how to prepare one or more flow deployments.    |
+| __[Deployment Management](/concepts/deployments-ux)__                       | A minimally opinionated set of files that describe how to prepare one or more flow deployments.    |
```

Comment:
Deployments vs. Deployment Management seems like a good way to frame the difference!

Reply:
🎉